autopilot/contractor/state.go

```diff
@@ -101,7 +98,14 @@ func (ctx *mCtx) WantedContracts() uint64 {
 }
 
 func (ctx *mCtx) SortContractsForMaintenance(contracts []contract) {
-	sortContractsForMaintenance(ctx.state.ContractsConfig(), contracts)
+	sort.SliceStable(contracts, func(i, j int) bool {
+		iUsable := contracts[i].Usability == api.ContractUsabilityGood
```

Comment:
Might as well add a `IsGood` helper on the contract to avoid the comparison all over the place.